src/ini/config.c

```diff
@@ -0,0 +1,78 @@
+#include "ini.h"
+
+typedef struct /*Can be changed in the future*/
+{
+        const char *orchestrator_ip;
+        const char *ochestrator_port;
+} configurationNode;
+
+typedef struct {
+        const char *port;
+        const char *expected_nodes;
+} configurationOrch;
+
+/*
+This function will be used in ini_parse() for node ini file parsing.
+If 0 return that means that the ini file is not in the right schem
+*/
+static int handler_node(void *user, const char *section, const char *name, const char *value) {
+
+        configurationNode *pconfig = (configurationNode *) user;
+
+#define MATCH(s, n) strcmp(section, s) == 0 && strcmp(name, n) == 0
```

Comment:
I still think this is not the place for macro definition. Definitely not inside the function with this indentation and with code duplication like we have here. Actually I think it should be placed in a header file, even a general one, since it can be used elsewhere, not only in `config.c`

Reply:
I agree with Mark.  This is ugly in several ways.  A C++ programmer will avoid macros altogether because there is no type checking.  Sometimes they are useful if they are general purpose, in which case they would go in a header.  In this case, the macro is written only to use a particular function argument.  I suggest skipping using this macro altogether, and just use plain strcmp().

Reply:
OK understood. I will change it.